lib/core/src/server/manager/babel-loader-manager.js

```diff
@@ -0,0 +1,26 @@
+import { includePaths } from '../config/utils';
+
+export default () => ({
+  test: /\.(mjs|jsx?)$/,
```

Comment:
Can we have `.tsx` files also taking into account? Because we use custom addon registered in `register.tsx`, to my mind it is good idea to support typescript as well.

Reply:
✔️